### DISCUSSION TITLE
Reqnroll is compatible with LivingDoc Azure extension

### DISCUSSION THREAD

**Discussion by tula-engineered-quality-hub** [reqnroll/Reqnroll#15] [Q&A] [answered]
Hi @gasparnagy! I have been working with Specflow since 2017, today I use the BDD process and LivingDoc is the output of this process. Is v1 of Reqnroll compatible with the LivingDoc cli and the LivingDoc extension on Azure? Count on me to use and highlight the tool's resources, I hope I can contribute to its evolution!

**Comment by gasparnagy** [comment]
The whole "LivingDoc" stuff is a closed-source implementation of Tricentis that does not have an extensible or documented API.

Please try it and see if it works, but I don't have big hopes. 😢

**Reply by tula-engineered-quality-hub** [reply]
Hi @gasparnagy! When i went to perform the test, i remembered that Specflow is a dependency on LivingDoc, but i still decided to see what would happen and, in fact, it doesn't compile the project with LivinDoc without Specflow.

I make myself available for any tests necessary regarding these tools and others that may arise. I have good expertise for such tests and contributions to the evolution of LivingDoc.

**Reply by gasparnagy** [reply, marked as answer]
Thx. Let's see what comes, but it seems for now we have to say that LivingDoc is not supported.

**Reply by maurice-hebben** [reply]
Hi @gasparnagy,
Also for LivingDoc, it would be great to have a possibity to publish the livingdoc.html on github actions (as a check or page). Initially LivingDoc was created to be compatible with AzureDevOps. Now many companies/projects are switching to github actions.
As an example, please see: https://github.com/dorny/test-reporter

**Reply by gasparnagy** [reply]
@maurice-hebben yeah... although as far as I understood GitHub Actions supports only Markdown for publishing these results, so we cannot really use any of the living doc features there. But I might be wrong on this.

**Comment by ThomasHeijtink** [comment]
Just, an update. We created a build task which generates the TestExecution.json file so that it "works" with Speclfow LivingDoc. I say works, because we are satisfied with just having the html published. But you can easily create a script or your own code if you want the test results to be included as well.

**Reply by gasparnagy** [reply]
@ThomasHeijtink Very good! Let's discuss about it at #68

**Reply by JohnathanSealMeridian** [reply]
Hello Folks,
Is there any update on Thomas's good work?
